hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/AttributeSearchService.kt

```diff
@@ -40,15 +42,18 @@ class AttributeSearchService {
     }
     getAllMatchers(request.matchers).forEach { validateMatcher(it) }
     getAllTextMatchers(request.matchers).forEach { it.validate() }
+    getAllBooleanMatchers(request.matchers).forEach { it.validate() }
     getAllIntegerMatchers(request.matchers).forEach { it.validate() }
     getAllDateMatchers(request.matchers).forEach { it.validate() }
+    getAllDateTimeMatchers(request.matchers).forEach { it.validate() }
```

Comment:
are there any fields that are date time?

Reply:
Just one - currentIncentive.dateTime. Might as well support the type 🤷